forum/models.py

```diff
@@ -214,27 +229,38 @@ def update_num_posts_on_post_insert(sender, instance, created, **kwargs):
             post.thread.last_post = post
             post.thread.save()
         invalidate_template_cache('latest_posts')
-    elif not created and post.moderation_state == "OK":
+    elif not created:
         post.thread.forum.set_last_post()
         post.thread.forum.save(update_fields=['last_post'])
         post.thread.set_last_post()
         post.thread.save(update_fields=['last_post'])
 
 
 @receiver(post_delete, sender=Post)
-def update_last_post_on_post_delete(sender, instance, **kwargs):
+def update_thread_on_post_delete(sender, instance, **kwargs):
     """Update num_posts counts and last_post pointers when a Post is deleted.
 
     Reduce num_posts by 1 for the author, forum, and thread
     Set last_post for the forum
     Set last_post for the thread. If this was the only remaining post
     in the thread and it was deleted, also delete the thread.
```

Comment:
What's the policy for ending sentences with/without a dot? I always struggle with this in code comments. Looks like if there's a single sentence in the line, then no need for dot. If multiple sentence in same line put dots at the end. But title is single sentence and has dot... Is this your recommendation?

Reply:
the policy is unclear. I like using . after the first sentence because it's normally used as a short description in html docs, and so is a good policy to get into. This specific example is a bit confusing because these appear to be list items (which I wouldn't normally put a dot after). I notice a few lines down I skip the . in the last sentence of a paragraph. It's all a mess, let's make some guidelines